File: corpus/2017q1/_p2q2_ok/_check/A+;0345.py
int q=0,t=0
Cardápio()
Pedidos(t)
def Pedidos(int t):
    double soma=0
    int l=-1,qtd=0
    while l!=0:
        print("Digite o código do pedido que deseja fazer :")
        l=int(input())
        if l!=0:
            if l==300:
                print("Digite a quantidade que deseja comprar :")
                qtd=int(input())
                soma=soma +(1.30*qtd)
            if l==301:
                print("Digite a quantidade que deseja comprar :")
                qtd=int(input())
                soma=soma +(1.40*qtd)
            if l==302:
                print("Digite a quantidade que deseja comprar :")
                qtd=int(input())
                soma=soma +(1.50*qtd)
            if l==303:
                print("Digite a quantidade que deseja comprar :")
                qtd=int(input())
                soma=soma +(1.40*qtd)
            if l==304:
                print("Digite a quantidade que deseja comprar :")
                qtd=int(input())
                soma=soma +(2.20*qtd)
            if l==305:
                print("Digite a quantidade que deseja comprar :")
                qtd=int(input())
                soma=soma +(1.00*qtd)
        qtd=0
    print("O valor da sua compra é :  "+soma)
    return soma
def Cardápio():
    for q=0 in range(0, 7):
        if q==0:
            for i=0 in range(0, 3):
                if i==0:
                    print(" Código ")
                    print("  ")
                if i==1:
                    print(" Item ")
                    print("  ")
                if i==2:
                    print(" Preço Unitário ")
                    print("  ")
            print("")
        if q==1:
            for i=0 in range(0, 3):
                if i==0:
                    print(" 300 ")
                    print("  ")
                if i==1:
                    print(" Pão de Queijo ")
                    print("  ")
                if i==2:
                    print(" R$ 1,30 ")
                    print("  ")
            print("")
        if q==2:
            for i=0 in range(0, 3):
                if i==0:
                    print(" 301 ")
                    print("  ")
                if i==1:
                    print(" Bauru ")
                    print("  ")
                if i==2:
                    print(" R$ 1,40 ")
                    print("  ")
            print("")
        if q==3:
            for i=0 in range(0, 3):
                if i==0:
                    print(" 302 ")
                    print("  ")
                if i==1:
                    print(" Cachorro Quente ")
                    print("  ")
                if i==2:
                    print(" R$ 1,50 ")
                    print("  ")
            print("")
        if q==4:
            for i=0 in range(0, 3):
                if i==0:
                    print(" 303 ")
                    print("  ")
                if i==1:
                    print(" Cheeseburguer ")
                    print("  ")
                if i==2:
                    print(" R$ 1,40 ")
                    print("  ")
            print("")
        if q==5:
            for i=0 in range(0, 3):
                if i==0:
                    print(" 304 ")
                    print("  ")
                if i==1:
                    print(" Suco ")
                    print("  ")
                if i==2:
                    print(" R$ 2,20 ")
                    print("  ")
            print("")
        if q==6:
            for i=0 in range(0, 3):
                if i==0:
                    print(" 305 ")
                    print("  ")
                if i==1:
                    print(" Refrigerante ")
                    print("  ")
                if i==2:
                    print(" R$ 1,00 ")
                    print("  ")
            print("")

File: corpus/2017q1/_p2q2_ok/_check/A+;0345.java

import java.util.Scanner;

public class P2E02 {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int q=0,t=0;
        Cardápio();
        Pedidos(t);

    }
    public static double Pedidos(int t){
        Scanner input = new Scanner(System.in);
        double soma=0;
        int l=-1,qtd=0;
        while (l!=0){
            System.out.println("Digite o código do pedido que deseja fazer :");
            l=input.nextInt();
            if (l!=0){
                if (l==300){
                    System.out.println("Digite a quantidade que deseja comprar :");
                    qtd=input.nextInt();
                    soma=soma+(1.30*qtd);
                }
                if (l==301){
                    System.out.println("Digite a quantidade que deseja comprar :");
                    qtd=input.nextInt();
                    soma=soma+(1.40*qtd);
                }
                if (l==302){
                   System.out.println("Digite a quantidade que deseja comprar :");
                    qtd=input.nextInt();
                    soma=soma+(1.50*qtd); 
                }
                if (l==303){
                    System.out.println("Digite a quantidade que deseja comprar :");
                    qtd=input.nextInt();
                    soma=soma+(1.40*qtd);
                }
                if (l==304){
                   System.out.println("Digite a quantidade que deseja comprar :");
                    qtd=input.nextInt();
                    soma=soma+(2.20*qtd); 
                }
                if (l==305){
                    System.out.println("Digite a quantidade que deseja comprar :");
                    qtd=input.nextInt();
                    soma=soma+(1.00*qtd);
                }
            }
            qtd=0;
        }
        System.out.println("O valor da sua compra é :  "+soma);
        return soma;

    }
    public static void Cardápio (){
        int L2[]= new int [2];
        int L3[]= new int [2];
        int L4[]= new int [2];
        int L5[]= new int [2];
        int L6[]= new int [2];
        int L7[]= new int [2];

        for (int q=0;q<7;q++){
                            if (q==0){
                                for (int i=0;i<3;i++){
                                    if (i==0){
                                        System.out.print(" Código ");
                                        System.out.print("  ");
                                    }
                                    if (i==1){
                                        System.out.print(" Item ");
                                        System.out.print("  ");
                                    }
                                    if (i==2){
                                        System.out.print(" Preço Unitário ");
                                        System.out.print("  ");
                                    }
                                }
                                System.out.println("");
                            }
                            if (q==1){
                                for (int i=0;i<3;i++){
                                   if (i==0){
                                        System.out.print(" 300 ");
                                        System.out.print("  ");
                                    }
                                    if (i==1){
                                        System.out.print(" Pão de Queijo ");
                                        System.out.print("  ");
                                    }
                                    if (i==2){
                                        System.out.print(" R$ 1,30 ");
                                        System.out.print("  ");
                                    }
                                }
                                System.out.println("");
                            }
                            if (q==2){
                               for (int i=0;i<3;i++){
                                    if (i==0){
                                        System.out.print(" 301 ");
                                        System.out.print("  ");
                                    }
                                    if (i==1){
                                        System.out.print(" Bauru ");
                                        System.out.print("  ");
                                    }
                                    if (i==2){
                                        System.out.print(" R$ 1,40 ");
                                        System.out.print("  ");
                                    }
                                }
                               System.out.println("");
                            }
                            if (q==3){
                               for (int i=0;i<3;i++){
                                    if (i==0){
                                        System.out.print(" 302 ");
                                        System.out.print("  ");
                                    }
                                    if (i==1){
                                        System.out.print(" Cachorro Quente ");
                                        System.out.print("  ");
                                    }
                                    if (i==2){
                                        System.out.print(" R$ 1,50 ");
                                        System.out.print("  ");
                                    }
                                }
                               System.out.println("");
                            }
                            if (q==4){
                               for (int i=0;i<3;i++){
                                    if (i==0){
                                        System.out.print(" 303 ");
                                        System.out.print("  ");
                                    }
                                    if (i==1){
                                        System.out.print(" Cheeseburguer ");
                                        System.out.print("  ");
                                    }
                                    if (i==2){
                                        System.out.print(" R$ 1,40 ");
                                        System.out.print("  ");
                                    }
                                }
                               System.out.println("");
                            }
                            if (q==5){
                               for (int i=0;i<3;i++){
                                    if (i==0){
                                        System.out.print(" 304 ");
                                        System.out.print("  ");
                                    }
                                    if (i==1){
                                        System.out.print(" Suco ");
                                        System.out.print("  ");
                                    }
                                    if (i==2){
                                        System.out.print(" R$ 2,20 ");
                                        System.out.print("  ");
                                    }
                                }
                               System.out.println("");
                            }
                            if (q==6){
                               for (int i=0;i<3;i++){
                                    if (i==0){
                                        System.out.print(" 305 ");
                                        System.out.print("  ");
                                    }
                                    if (i==1){
                                        System.out.print(" Refrigerante ");
                                        System.out.print("  ");
                                    }
                                    if (i==2){
                                        System.out.print(" R$ 1,00 ");
                                        System.out.print("  ");
                                    }
                                }
                               System.out.println("");
                            }

        }
    }
}
